blockonomics-woocommerce.php

```diff
@@ -464,24 +464,31 @@ function bnomics_async_scripts($url)
 add_action('admin_notices', 'blockonomics_plugin_activation');
 
 global $blockonomics_db_version;
-$blockonomics_db_version = '1.1';
+$blockonomics_db_version = '1.2';
 
 function blockonomics_create_table() {
-    // Create blockonomics_orders table
+    // Create blockonomics_payments table
     // https://codex.wordpress.org/Creating_Tables_with_Plugins
     global $wpdb;
     global $blockonomics_db_version;
+    /* for db version 1.2, new table blockonomics_payments is introduced
+        status is renamed to payment_status
+        satoshi is renamed to expected_satoshi
+        value is renamed to expected_fiat & it is no longer longtext
+        2 new fields are added for logging purpose only - paid_satoshi & paid_fiat */ 
 
-    $table_name = $wpdb->prefix . 'blockonomics_orders';
+    $table_name = $wpdb->prefix . 'blockonomics_payments';
     $charset_collate = $wpdb->get_charset_collate();
     $sql = "CREATE TABLE $table_name (
         order_id int NOT NULL,
-        status int NOT NULL,
+        payment_status int NOT NULL,
         crypto varchar(3) NOT NULL,
         address varchar(191) NOT NULL,
-        satoshi int,
+        expected_satoshi int,
+        expected_fiat double,
         currency varchar(3),
-        value longtext,
+        paid_satoshi int,
```

Comment:
@shivaenigma, small note regarding the table structure: We should probably use BigInt for satoshi values instead of Int, although this will only affect really large payments.